package.json

```diff
@@ -28,7 +28,7 @@
   "dependencies": {
     "ember-cli-babel": "^7.11.1",
     "ember-cli-htmlbars": "^4.0.0",
-    "gridstack": "^0.6.2",
+    "gridstack": "^0.6.3",
```

Comment:
Hey glad you guys and updating and keeping up with the latest rev! Yeah.
one thing to noticed is that your package does NOT need to list jquery and jquery-ui as dependencies - I will bring those in my package (there is work to remove that lib as well in the future).
Also **gridstack-all.min.js** will now include a **much smaller jqeury-ui** so you shouldn't include it anyway.

Reply:
Thanks for the hint!